### PR TITLE
Crypto

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Renames KAZkoin to more generic Koin alongside adding icons to it
	
<hr>
</details>

## Changelog
:cl:
tweak: more generic crypto
imageadd: BonziMoney program icons
/:cl:

### CODE REVIEW

File: code/modules/modular_computers/file_system/programs/generic/coin_miner.dm [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+	filename = "Zipped K-oin"
 	filedesc = "Key-Authenticated Zipped K-oin"
 	nanomodule_path = /datum/nano_module/program/coin_miner
-	program_icon_state = "supply"
+	program_icon_state = "crypto"
 	program_key_state = "rd_key"
-	program_menu_icon = "cart"
-	extended_desc = "A endlessly compressed file that when processed turns into valueable credit on the non-solfed market. The bigger the file, the more its worth."
+	program_menu_icon = "home"
+	extended_desc = "An endlessly compressed file that when processed turns into valueable credit on the terran market. The bigger the file, the more its worth."
```
Comment (ThePainkiller) [inline code/modules/modular_computers/file_system/programs/generic/coin_miner.dm:8]:
```suggestion
	extended_desc = "An endlessly compressed file that when processed turns into valuable credit on the intergalactic market. The bigger the file, the more its worth."
```

File: code/modules/modular_computers/file_system/programs/generic/coin_miner.dm [hunk 2]
```diff
@@ -1,11 +1,11 @@
 /datum/computer_file/program/coin_miner
-	filename = "KAZipped K-oin"
+	filename = "Zipped K-oin"
 	filedesc = "Key-Authenticated Zipped K-oin"
```
Comment (ThePainkiller) [inline code/modules/modular_computers/file_system/programs/generic/coin_miner.dm:3]:
```suggestion
	filedesc = "Zipped K-oin decoder"
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

**Comment by Trilbyspaceclone** [comment]
get the sprites approved by Giget

File: code/modules/modular_computers/file_system/programs/generic/coin_miner.dm [hunk 3]
```diff
@@ -1,11 +1,11 @@
 /datum/computer_file/program/coin_miner
-	filename = "KAZipped K-oin"
+	filedesc = "Zipped K-oin decoder"
```
Comment (Trilbyspaceclone) [inline code/modules/modular_computers/file_system/programs/generic/coin_miner.dm:2]: two file desc?

**Comment by AysheDaArtist** [comment]
I'm alright with this for now, but everytime I see it, it's going to be glaring reminder to get it fixed up immediately

I'd prefer some kind of Bonzi Buddy alternative that was more snow themed or lore themed

**Comment by ThePainkiller** [comment]
> I'd prefer some kind of Bonzi Buddy alternative that was more snow themed or lore themed

A Yeti instead of a monkey, perhaps?

**Comment by Trilbyspaceclone** [comment]
sorry the sprites are to goofy

**Pull request closed without merging**